architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,32 +234,55 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
     - created: 2016-01-29T13:40:11Z
       dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
       generation: 1
-      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+      image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+    tag: latest <4>
 ----
+
+<1> The name of the image stream. 
+<2> The image source (repository or other image stream).
+<3> The name of the external image to use.
+<4> The image tag.
+
 ====
 
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+* You can trigger Builds and Deployments when new image is pushed to the registry, or schedule the import to do it later.
+* You can share images using fine-grained access and quickly distribute images across your teams.
+* If the source image changes, the Image Stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+However, if an image author pushes a newer version of an image without properly tagging the image, it can accidentally create a new distinct image (with newer version information). 
+This condition can impact applications in unexpected ways.
```

Comment:
this needs to elaborate on what it means to "push an image without properly tagging it" (and how/why that is problem since just above we claimed the imagestream insulates you from this)